src/ethereum_spec_tools/evm_tools/t8n/t8n_types.py

```diff
@@ -339,18 +339,10 @@ def json_encode_receipts(self) -> Any:
 
             receipt_dict["gasUsed"] = hex(receipt.cumulative_gas_used)
             receipt_dict["bloom"] = "0x" + receipt.bloom.hex()
-            receipt_dict["logs"] = [
-                {
-                    "address": "0x" + log.address.hex(),
-                    "topics": ["0x" + topic.hex() for topic in log.topics],
-                    "data": "0x" + log.data.hex(),
-                }
-                for log in receipt.logs
-            ]
```

Comment:
Why were these removed?

Reply:
Because `fill` expects fields in the log aren't directly part of the log (e.g. blockhash). It is also unclear how these fields are being used by `fill`. It doesn't complain if the log is not supplied by the `t8n` result.

Removing the logs from the result is more of a quick fix. In the long term, we should analyse which fields are actually essential and work towards including them.

Reply:
I think you mentioned that they weren't there previously? So we'd never noticed that fields were missing? I.e., removing them now is not really taking anything away that was there previously?